File: Cuaderno_1_JAVA/ej5.java
public class ej5 {
    public static void main(String[] args) {
        int[] hora = { 15, 23, 28 };
        int segundosActual = hora[0] * 3600 + hora[1] * 60 + hora[2];
        int segundosMediaNoche = 24 * 3600;

        int segundosAMediaNoche = segundosMediaNoche - segundosActual;
        System.out.println("Llevamos: " + segundosActual + " segundos de día. Quedan " + segundosAMediaNoche
                + " segundos para media noche");
    }
}

File: Cuaderno_1/ej5.py
hora = [15, 23, 18] #Horas, minutos, segundos

segundosActual = hora[0] * 3600 + hora[1] * 60 + hora[2]

segundosMediaNoche = 24 * 3600

segundosAMediaNoche = segundosMediaNoche - segundosActual

print(f"Llevamos {segundosActual} segundos de día. Quedan {segundosAMediaNoche} segundos para media noche")
